node_agent/main.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 Istio Authors
```

Comment:
@lookuptable , agreed we should move the structure related discussion to a dedicated issue. Since you won't be back until next week, we can have an in-person discussion to reach an agreement then. And do code-refactor if necessary.

In the meantime, let's focus on everything else so Sarvani won't get blocked. Thanks.